pkg/code-generator/codegen/generator.go

```diff
@@ -75,12 +71,7 @@ func GenerateFiles(project *model.Project, skipOutOfPackageFiles, skipGeneratedT
 		}
 		files = append(files, fs...)
 	}
-	for i, v := range files {
```

Comment:
This PR consolidates writing content and headers into a single Writer type. Therefore, this code can be removed